scripts/go-function-push.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+
+set -ex
+
+image_tag=${TAG:-dev}
+prefix="release-go-functions-"
+[[ "${image_tag}" = "${prefix}"* ]] && image_tag="${image_tag#$prefix}"
+image_name=gcr.io/kpt-functions/"${CURRENT_FUNCTION}"
+image="${image_name}":"${image_tag}"
+docker push "${image_name}:latest"
```

Comment:
No "latest" please, we should use "unstable" instead.
Please check the "versioning" section in the doc.